frontend/App.vue

```diff
@@ -1,15 +1,5 @@
 <template>
-    <div>
-        <nav>
-            <router-link class="router-link" to="/">Home</router-link>
-            <router-link class="router-link" to="/about">About</router-link>
-            <router-link class="router-link" to="/profile">Profile</router-link>
-            <router-link class="router-link" to="/manage">Manage</router-link>
```

Comment:
Let's make sure we get these hooked up soon!
When #865 is merged, can we add a link from `/profile` into `/manage` just hardcoded for now?